### ISSUE TITLE
Identify list of unit, places, times ontologies (ISO if possible)

### ISSUE THREAD

**Issue by FrancisJMR** [goodlyrottenapple/adam2-demo#13]
Identify list of unit, places, times ontologies (ISO if possible)

**Comment by goodlyrottenapple** [comment]
found this for countries: https://www.w3.org/Consortium/Offices/Presentations/RDFTutorial/rdfs/Countries.owl

**Comment by goodlyrottenapple** [comment]
ended up using ISO 3166-2 for countries with the label 'ISO 3166-2:GB' for the UK, for example

**Comment by FrancisJMR** [comment]
Thanks saw the new updates with countries and dates in ISO. Plus looks like you added widgets to show flags and calendar. Are these owl sources or did you have to format them?

**Comment by goodlyrottenapple** [comment]
no, the countries are adapted from another library with the iso codes being added manually.
